app-frontend/config/webpack/global.js

```diff
@@ -1,209 +1,209 @@
+                                                'css-loader!postcss-loader')
+            }, {
+                test: /\.(scss|sass)$/,
+                loader: DEVELOPMENT ? 'style-loader!' + stylesLoader
+                    : ExtractTextPlugin.extract('style-loader', stylesLoader)
+            }, {
+                test: /\.(woff2|woff|ttf|eot|svg)(\?[0-9]+)?(#[0-9a-zA-Z]+)?$/,
+                loaders: [
+                    'url-loader?name=assets/fonts/[name]_[hash].[ext]'
+                ]
+            }, {
+                test: /\.(jpe?g|png|gif)$/i,
+                loaders: [
+                    'url-loader?name=assets/images/[name]_[hash].[ext]&limit=10000!image-webpack'
+                ]
+            }, {
+                test: /\.(m4v|ogg|webm)$/i,
+                loaders: [
+                    'url-loader?name=assets/video/[name]_[hash].[ext]&limit=10000'
+                ]
+            }, {
+                test: require.resolve('angular-deferred-bootstrap'),
+                loaders: [
+                    'expose?deferredBootstrapper'
+                ]
+            }, {
+                test: require.resolve('angular'),
+                loaders: [
+                    'expose?angular'
+                ]
+            }, {
+                test: require.resolve('jquery'),
+                loaders: [
+                    'expose?$',
+                    'expose?jQuery'
+                ]
+            }, {
+                test: require.resolve('leaflet'),
+                loaders: [
+                    'expose?L'
+                ]
+            }, {
+                test: require.resolve('jointjs'),
+                loaders: [
+                    'expose?joint'
+                ]
+            }, {
+                test: require.resolve('moment'),
+                loaders: [
+                    'expose?moment'
+                ]
+            }, {
+                test: /node_modules[\\\/]auth0-lock[\\\/].*\.js$/,
+                loaders: ['transform-loader/cacheable?brfs',
+                          'transform-loader/cacheable?packageify']
+            }, {
+                test: /node_modules[\\\/]auth0-lock[\\\/].*\.ejs$/,
+                loader: 'transform-loader/cacheable?ejsify'
+            }, {
+                test: /\.json$/,
+                loader: 'json'
+            }]
+        },
+
+        // post css
+        postcss: [autoprefixer({browsers: ['last 5 versions']})],
+
+        imageWebpackLoader: {
+            pngquant: {
+                quality: '66-90',
+                speed: 4
+            }
+        },
+
+        eslint: {
+            configFile: './.eslintrc'
+        },
+
+        // load plugins
+        plugins: [
+            new webpack.ProvidePlugin({
+                $: 'jquery',
+                jQuery: 'jquery',
+                L: 'leaflet'
+            }),
+            new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
+            new webpack.optimize.DedupePlugin(),
```

Comment:
It works! There were a couple indentation warnings being thrown by the linter, and I verified that fixing them caused a webpack crash. Making this change allowed me to fix the warnings without a crash.

But wait, there's more!
It _also_ seems to have fixed @designmatty 's crash on font updates. :tada: :tada: :tada: